identity_stardust/examples/create_did.rs

```diff
@@ -28,79 +35,46 @@ use identity_stardust::StardustDID;
 use identity_stardust::StardustDocument;
 use identity_stardust::StardustVerificationMethod;
 
-// PROBLEMS SO FAR:
-// 1) Alias Id is inferred from the block, so we have to use a placeholder DID for creation.
-// 2) Cannot get an Output Id back from an Alias Id (hash of Output Id), need to use Indexer API.
-// 3) The Output response from the Indexer is an Output, not a Block, so cannot infer Alias ID from it (fine since we
-// use the ID to retrieve the Output in the first place).    The OutputDto conversion is annoying too.
-// 4) The pieces needed to publish an update are fragmented (Output ID for input, amount, document), bit annoying to
-// reconstruct.    Use a holder struct like Holder { AliasOutput, StardustDocument } with convenience functions?
-// 5) Inferred fields such as the controller and governor need to reflect in the (JSON) Document but excluded from the
-// StardustDocument serialization when published.    Handle with a separate `pack` function like before?
+static ENDPOINT: &str = "https://api.testnet.shimmer.network/";
+static FAUCET_URL: &str = "https://faucet.testnet.shimmer.network/api/enqueue";
 
 /// Demonstrate how to embed a DID Document in an Alias Output.
-///
-/// iota.rs alias example:
-/// https://github.com/iotaledger/iota.rs/blob/f945ccf326829a418334942ae9cf53b8fab3dbe5/examples/outputs/alias.rs
-///
-/// iota.js mint-nft example:
-/// https://github.com/iotaledger/iota.js/blob/79a71d3a2ad03be5bd6148689d083947f3b98476/packages/iota/examples/mint-nft/src/index.ts
-#[tokio::main]
-async fn main() -> anyhow::Result<()> {
-  // let endpoint = "http://localhost:14265";
-  let endpoint = "https://api.alphanet.iotaledger.net";
-  let faucet_manual = "https://faucet.alphanet.iotaledger.net";
-  let network_name = NetworkName::try_from("alpha")?;
-
-  // ===========================================================================
-  // Step 1: Create or load your wallet.
-  // ===========================================================================
-
-  // let keypair = identity_core::crypto::KeyPair::new(identity_core::crypto::KeyType::Ed25519).unwrap();
-  // println!("PrivateKey: {}", keypair.private().to_string());
-  // let mnemonic =
-  // iota_client::crypto::keys::bip39::wordlist::encode(keypair.private().as_ref(),&bip39::wordlist::ENGLISH).unwrap();
-
-  // NOTE: this is just a randomly generated mnemonic, REMOVE THIS, never actually commit your seed or mnemonic.
-  let mnemonic = "veteran provide abstract express quick another fee dragon trend extend cotton tail dog truly angle napkin lunch dinosaur shrimp odor gain bag media mountain";
-  println!("Mnemonic: {}", mnemonic);
-  let secret_manager = SecretManager::Mnemonic(MnemonicSecretManager::try_from_mnemonic(mnemonic)?);
-
-  // Create a client instance.
-  let client = Client::builder()
-    .with_node(endpoint)?
-    .with_node_sync_disabled()
-    .finish()?;
+pub async fn run() -> anyhow::Result<(Client, Address, SecretManager, StardustDocument)> {
```

Comment:
Agreed. The only issue is that then we can't write `mod 0_create` in the other examples to depend on the create example, since identifiers in Rust cannot start with digits.